systemtest/rum_test.go

```diff
@@ -87,8 +87,11 @@ func TestRUMAllowServiceNames(t *testing.T) {
 	srv := apmservertest.NewUnstartedServer(t)
 	srv.Config.AgentAuth.SecretToken = "abc123"
 	srv.Config.RUM = &apmservertest.RUMConfig{
-		Enabled:           true,
-		AllowServiceNames: []string{"allowed"},
+		Enabled: true,
+	}
+	srv.Config.AgentAuth.Anonymous = &apmservertest.AnonymousAuthConfig{
+		Enabled:      true,
+		AllowService: []string{"allowed"},
```

Comment:
I've noticed that I needed to set `apm-server.auth.anonymous.enabled=true` in order for `AllowService` to work, while I understand this is intended, should we note this somewhere in our docs? It seems to deviate from what was previously required, just enabling `rum.enabled: true` and `rum.allow_service: [a, b]?

Reply:
Sounds good, can you please open a docs issue for that?